ansible/roles/ssc/defaults/main.yml

```diff
@@ -0,0 +1,34 @@
+# SSC Configuration
+ssc_enabled: false
+ssc_version: "0.5.0-alpha.2"
+ssc_image: "sagaxyz/ssc:{{ ssc_version }}"
+ssc_namespace: "sagasrv-ssc"
+ssc_rpc_port: 26657
+ssc_p2p_port: 26656
+ssc_grpc_port: 9090
+ssc_metrics_port: 26660
+
+# SSC Resources
+ssc_resources:
+  requests:
+    memory: "512Mi"
+    cpu: "0.2"
+  limits:
+    memory: "1Gi"
+    cpu: "0.5"
+
+# SSC Storage
+ssc_storage_class: ""
```

Comment:
We should remove this. All the cluster require a `saga-default` storage class. That's a fair assumption. I don't see a case where we should override that. I'd just hardcode it to `saga-default`